dev/com.ibm.ws.microprofile.config.1.4_fat/fat/src/com/ibm/ws/microprofile/config14/test/CacheForTest.java

```diff
@@ -0,0 +1,80 @@
+package com.ibm.ws.microprofile.config14.test;
+
+import java.io.File;
+
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.runner.RunWith;
+
+import com.ibm.websphere.simplicity.ShrinkHelper;
+import com.ibm.ws.microprofile.config.fat.repeat.RepeatConfigActions;
+import com.ibm.ws.microprofile.config14.configAccessor.cacheFor.web.CacheForServlet;
+
+import componenttest.annotation.Server;
+import componenttest.annotation.TestServlet;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.rules.repeater.RepeatTests;
+import componenttest.topology.impl.LibertyServer;
+import componenttest.topology.utils.FATServletClient;
+
+/**
+ * Example Shrinkwrap FAT project:
+ * <li> Application packaging is done in the @BeforeClass, instead of ant scripting.
+ * <li> Injects servers via @Server annotation. Annotation value corresponds to the
+ * server directory name in 'publish/servers/%annotation_value%' where ports get
+ * assigned to the LibertyServer instance when the 'testports.properties' does not
+ * get used.
+ * <li> Specifies an @RunWith(FATRunner.class) annotation. Traditionally this has been
+ * added to bytecode automatically by ant.
+ * <li> Uses the @TestServlet annotation to define test servlets. Notice that no @Test
+ * methods are defined in this class. All of the @Test methods are defined on the test
+ * servlet referenced by the annotation, and will be run whenever this test class runs.
+ */
+@RunWith(FATRunner.class)
+public class CacheForTest extends FATServletClient {
+
+    public static final String APP_NAME = "configAccessorCacheForApp";
+    public static final String SERVER_NAME = "Config14Server";
+
+    @ClassRule
+    public static RepeatTests r = RepeatConfigActions.repeatConfig14(SERVER_NAME);
+
+    @Server(SERVER_NAME)
+    @TestServlet(servlet = CacheForServlet.class, contextRoot = APP_NAME)
+    public static LibertyServer server;
+
+    @BeforeClass
+    public static void setUp() throws Exception {
+        // Create a WebArchive that will have the file name 'app1.war' once it's written to a file
+        // Include the 'app1.web' package and all of it's java classes and sub-packages
+        // Automatically includes resources under 'test-applications/APP_NAME/resources/' folder
+        // Exports the resulting application to the ${server.config.dir}/apps/ directory
+        WebArchive war = ShrinkWrap.create(WebArchive.class, APP_NAME + ".war")
+                        .addPackages(true, "com.ibm.ws.microprofile.config14.configAccessor")
+                        .addAsManifestResource(new File("test-applications/" + APP_NAME + "/META-INF/microprofile-config.properties"),
+                                               "microprofile-config.properties");
+
+        ShrinkHelper.exportDropinAppToServer(server, war);
```

Comment:
Please add `SERVER_ONLY` as a deployment option here to avoid the same app being deployed and started for other tests which use the same server.

`SERVER_ONLY` instructs the test framework not to add the app to the `publish` directory and only deploy it to the server.